ports/tinytiff/portfile.cmake

```
@@ -0,0 +1,37 @@
+vcpkg_check_features(
+    OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    FEATURES
+        winapi      TinyTIFF_USE_WINAPI_FOR_FILEIO
+)
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO jkriege2/TinyTIFF
+    REF ${VERSION}
+    SHA512 28fb3d1ef1630a4d20da021ccca93f99df8bd29462525be312dfb028239176ca940a43407b2db10488d891a1fbca65d8a59bc6cc097765389f35021e8b423885
+    HEAD_REF master
+)
+
+vcpkg_cmake_configure(
+    SOURCE_PATH "${SOURCE_PATH}"
+    OPTIONS
+        ${FEATURE_OPTIONS}
+        -DTinyTIFF_BUILD_TESTS=OFF
+    PATCHES
+        # without this patch, the MSVC compiler will crash during the build process 
+        "msvc-support.patch"
```

```suggestion
        "msvc-support.patch" # without this patch, the MSVC compiler will crash during the build process
```
If this patch is not for vcpkg, then there needs to be a report.

---

@FrankXie05 Thanks for the review. 

 This patch is necessary to resolve a compatibility issue with the MSVC compiler and winapi feature. 

I have also submitted a pull request (jkriege2/TinyTIFF#30), which reports about this issue.